doc/website/advanced/configuration-guide.md

```diff
@@ -142,12 +169,19 @@ size = 1024
 count = 100
 ```
 
-When no config file is specified, a hard-coded version similar to the [default config](https://github.com/eclipse-iceoryx/iceoryx/blob/master/iceoryx_posh/etc/iceoryx/roudi_config_example.toml) will be used.
+When no configuration file is specified a hard-coded version similar to the 
+[default config](https://github.com/eclipse-iceoryx/iceoryx/blob/master/iceoryx_posh/etc/iceoryx/roudi_config_example.toml)
```

Comment:
```suggestion
[default config](https://github.com/eclipse-iceoryx/iceoryx/blob/v2.0.0/iceoryx_posh/etc/iceoryx/roudi_config_example.toml)
```

Reply:
Since it is merged to master I would leave it has it is.